src/metamask/confirmTransaction.ts

```diff
@@ -5,6 +5,8 @@ import { clickOnButton, typeOnInputField } from '../helpers';
 
 import { GetSingedIn } from './index';
 
+const MIN_GAS = 21000;
```

Comment:
why is minimal gas hardcoded to fixed value?

Reply:
That is the value enforced by metamask (and ethereum ), lower than that Metamask validation will disable the Save button and it also makes sense because tx cannot execute.
![image](https://user-images.githubusercontent.com/8836210/191924952-6571b75f-57f6-4726-baee-f1b6d7342675.png)

Reply:
then we should do additional research on what is a case of other `EVM` chains

Reply:
It doesn't really matter as metamask won't allow lower value anyways haha